jp2_pc/Source/Lib/Renderer/Primitives/FastBump.cpp

```diff
@@ -1239,8 +1239,10 @@ void ApplyCurves(rptr<CMesh> pmsh)
 
 			// did we get a bump map from the above cast
 			Assert(pbump);
-			if (!pbump)
-				return;
+			if (!pbump) {
+				pi.pmpPolygon()->pmx3ObjToTexture = SwapMatrices.pmx3GetMatrix(pi.d3Normal());
+				continue;
+			}
 
```

Comment:
It was wrong to just return, we are in a loop. That matrix is required to exist elsewhere, so we ensure that it exists.